gradio/utils.py

```diff
@@ -680,3 +681,8 @@ def validate_url(possible_url: str) -> bool:
 
 def is_update(val):
     return type(val) is dict and "update" in val.get("__type__", "")
+
+
+def make_valid_filename(name):
```

Comment:
We already have `utils.strip_invalid_filename_characters()`. Can we use that insetad?